aarch64/src/interrupts.S

```diff
@@ -0,0 +1,170 @@
+.section .text
```

Comment:
I think for the filename, I'd default to whatever plan9 used.

Reply:
I think a lot ends up in l.S or trap.c

Reply:
Went for trap.S and trap.rs.  A bit of overloading, but unless you have a better suggestion, I'll stick with it.